### PR TITLE
Revise installation infrastructure

### PR DESCRIPTION
**Pull request by tashrifbillah**
Hi @RyanZurrin , can you review this PR? The installation instruction that you gave is compiled here in modified form:

https://github.com/pnlbwh/HD-BET/wiki/Installation-instruction-for-PNL

I have tested it a lot. If interested, you can test it using the following commands. No additional command is necessary.

```
cd /rfanfs/pnl-zorro/projects/DIAGNOSE_CTE_U01/derivatives/pnlpipe/sub-1003/ses-01/anat
/software/rocky9/miniconda3/envs/hd-bet/bin/hd-bet -i sub-1003_ses-01_desc-Xc_T1w.nii.gz -o sub-1003_ses-01_desc-Xc_mask.nii.gz -mode accurate -b 0
```

After you review, we shall remove the above private paths from this description.

### CODE REVIEW

File: requirements.txt
Comment (RyanZurrin) [inline requirements.txt:1]: Do we not need to use the `conda install pytorch torchvision` command anymore with these changes?

**Review by RyanZurrin — COMMENTED** [review]
Should we not include link to the updated instrucions in the main readme?

These updated instructions would not only apply to PNL, but to anyone wanting to use this on new gpus, no?

It seems like the instructions for the updated env setup are not easy to find. Most poeple assume the proper instrucitons or info will be in the main readme. At least that is what I assume.

**Comment by tashrifbillah** [comment]
I know but if we do, then we lose ability to request pull to the main repository. We can instead make a `pnl` branch and include instructions in that branch without modifying the master.

**Comment by RyanZurrin** [comment]
Ahh, okay I was overlooking that fact. I think your idea for a pnl branch is a good solution.

**Review by RyanZurrin — APPROVED** [review]

**Review by RyanZurrin — APPROVED** [review]

**Comment by tashrifbillah** [comment]
Hi Ryan, although I suggested `pnl` name before, can we come up with a more inclusive/indicative name? How about `modern-gpus`? Please comment either way.

**Comment by RyanZurrin** [comment]
Hello Tahsrif, I think modern-gpus is more descriptive and is a good idea. It lets anyone know that it's not just a for pnl but for anyone who wants to run this program on modern gpus.

**Comment by tashrifbillah** [comment]
Thank you Ryan for your comment. `modern-gpus` branch is now the default. This task is now complete. We shall deploy it across PNL platforms in near future.

**Pull request merged**